src/components/DataDecorator/DataDecorator.js

```diff
@@ -263,6 +281,14 @@ DataDecorator.propTypes = {
 
   /** @type {func} Descriptive text for screen readers that details the severity of a score. */
   scoreDescription: PropTypes.func,
+
+  /** @type {object} Mid-line truncation options applied to value of decorator if applicable. */
+  midLineTruncation: PropTypes.shape({
+    enabled: PropTypes.bool,
```

Comment:
Is it possible to remove the need for an `enabled` or is there a use case for it? Can it be implied when `back`, `front`, and `maxLength` exist?